statsmodels/stats/tests/test_diagnostic.py

```diff
@@ -463,6 +463,34 @@ def test_compare_lr(self):
         assert_almost_equal(wt[1], waldtest['pvalue'], decimal=11)
 
 
+    def test_spec_white(self):
+        resdir = os.path.join(cur_dir, "results")
+        wsfiles = ['wspec1.csv', 'wspec2.csv', 'wspec3.csv', 'wspec4.csv']
+        for file in os.listdir(resdir):
```

Comment:
I think this os.listdir can be skipped
just loop over wsfiles directly

Reply:
I chose the method shown more out of fault tolerance than any performance concerns but now that I think about it if someone accidentally removes one of the test files the correct action is to error out. I will change it in the next commit.